hyperapp.d.ts

```diff
@@ -0,0 +1,40 @@
+export as namespace hyperapp
+
+// VirtualNode part
+
+export interface VirtualNode<Data> {
+  tag: string
+  data?: Data
+  children: VirtualElement<{}>[]
+}
+
+export type VirtualElement<Data> = VirtualNode<Data> | string
+
+export interface Component<Data> {
+  (data?: Data, ...children: VirtualElement<{}>[]): VirtualNode<Data>
+}
+
+export function h<Data>(
+  tag: Component<Data> | string,
+  data?: Data,
+  children?: VirtualElement<{}>[] | VirtualElement<{}> | number
```

Comment:
Without it TSC will yield an exception if I want to use `h()` that way
```tsx
h('div', null, h('span', null, 'Hi.'))
```

This simple thing will not be possible to
```tsx
h('div', null, 'Hi.')

// ask for
h('div', null, ['Hi.'])
```

Do you really want to remove features by typing ?